 docker_image/snowshu.dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.7
+
+RUN pip3 install snowshu>=0.0.3 && \
```

Comment:
Not sure if there is a more convenient way of specifying the version of snowshu to install, but as long as the image is built after the push to pypi, this should pull the latest version in.

Might be worth a quick investigation into what types of parameterization github actions supports for docker builds.

Reply:
Since the latest version is what we want we just need to perform `pip3 install snowshu`
https://docs.python.org/3/installing/index.html#basic-usage